ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryControllerTest.java

```diff
@@ -395,16 +413,10 @@ public void peerSeenTwice() throws InterruptedException {
         .send(eq(peers.get(1)), matchPacketOfType(PacketType.PING));
 
     // Simulate a PONG message from peer[0].
-    final PongPacketData packetData =
+    final PongPacketData packetData0 =
         PongPacketData.create(localPeer.getEndpoint(), mockPacket.getHash());
-    Packet pongPacket = Packet.create(PacketType.PONG, packetData, keyPairs.get(0));
-    controller.onMessage(pongPacket, peers.get(0));
-
-    // Simulate a NEIGHBORS message from peer[0] listing peer[2].
-    final NeighborsPacketData neighbors =
-        NeighborsPacketData.create(Collections.singletonList(peers.get(2)));
-    Packet neighborsPacket = Packet.create(PacketType.NEIGHBORS, neighbors, keyPairs.get(0));
-    controller.onMessage(neighborsPacket, peers.get(0));
+    final Packet pongPacket0 = Packet.create(PacketType.PONG, packetData0, keyPairs.get(0));
+    controller.onMessage(pongPacket0, peers.get(0));
```

Comment:
This is the same as the section above as well.  Looks like we could use a `respondWithPong(DiscoveryPeer, KeyPair)` method.